src/get_started.md

```diff
@@ -26,34 +25,32 @@ The following complete `index.html` webxdc app example shows an input field and
       function receiveUpdate(update) {
         document.getElementById('output').innerHTML += update.payload + "<br>";
       }
-    
       window.webxdc.setUpdateListener(receiveUpdate, 0);
     </script>
   </body>
 </html>
 ```
 
-Now zip the directory containing the `index.html`:
+To package the app as a `.xdc` file, zip the directory containing the `index.html`:
 
 ```shell
 zip -9 --recurse-paths "myapp.xdc" PATH_TO_DIR
 ```
-You can now share the resulting `myapp.xdc` file in a chat and everyone can hit "start" to start our little "chat app". 
 
-However, for faster and more convenient development round trips we recommend to try using the minimal [hello webxdc development](https://github.com/webxdc/hello) tool.
+Now it's possible to share the `myapp.xdc` file in any chat—recipients can hit "Start" to begin using the app to enter text in this input field and send updates to each other. 
```

Comment:
dividing a sentence by a dash without spaces is very strange, at lest in german. i am reading "chat-recipients" and i am just wondering :)

Reply:
It's an emdash—often used to break clauses—and it may appear as a small dash in git.

Reply:
i know - but it looks still strange to me as a non-english native. maybe it is only me, however, to have easy language, it can be avoided easily.

Reply:
Be welcome to change it for what feels comfortable.

Reply:
Actually, I'll just change it to a colon.